integrations/server_test.go

```diff
@@ -0,0 +1,519 @@
+			fields: fields{
+				Users: []chronograf.User{
+					{
+						ID:         1, // This is artificial, but should be reflective of the users actual ID
+						Name:       "billibob",
+						Provider:   "github",
+						Scheme:     "oauth2",
+						SuperAdmin: true,
+						Roles: []chronograf.Role{
+							{
+								Name:         "admin",
+								Organization: "0",
+							},
+						},
+					},
+				},
+			},
+			args: args{
+				server: &server.Server{
+					GithubClientID:     "not empty",
+					GithubClientSecret: "not empty",
+				},
+				method: "GET",
+				path:   "/chronograf/v1/users",
+				principal: oauth2.Principal{
+					Organization: "0",
+					Subject:      "billibob",
+					Issuer:       "github",
+					ExpiresAt:    time.Now().Add(10 * time.Second),
+					IssuedAt:     time.Now(),
+				},
+			},
+			wants: wants{
+				statusCode: 200,
+				body: `
+					{
+					  "links": {
+					    "self": "/chronograf/v1/users"
+					  },
+					  "users": [
+					    {
+					      "links": {
+					        "self": "/chronograf/v1/users/1"
+					      },
+					      "id": "1",
+					      "name": "billibob",
+					      "provider": "github",
+					      "scheme": "oauth2",
+					      "superAdmin": true,
+					      "roles": [
+					        {
+					          "name": "admin",
+					          "organization": "0"
+					        }
+					      ]
+					    }
+					  ]
+					}`,
+			},
+		},
+		{
+			name:    "GET /users",
+			subName: "Single User in the Default Organization as Admin",
+			fields: fields{
+				Users: []chronograf.User{
+					{
+						ID:       1, // This is artificial, but should be reflective of the users actual ID
+						Name:     "billibob",
+						Provider: "github",
+						Scheme:   "oauth2",
+						Roles: []chronograf.Role{
+							{
+								Name:         "admin",
+								Organization: "0",
+							},
+						},
+					},
+				},
+			},
+			args: args{
+				server: &server.Server{
+					GithubClientID:     "not empty",
+					GithubClientSecret: "not empty",
+				},
+				method: "GET",
+				path:   "/chronograf/v1/users",
+				principal: oauth2.Principal{
+					Organization: "0",
+					Subject:      "billibob",
+					Issuer:       "github",
+					ExpiresAt:    time.Now().Add(10 * time.Second),
+					IssuedAt:     time.Now(),
+				},
+			},
+			wants: wants{
+				statusCode: 200,
+				body: `
+					{
+					  "links": {
+					    "self": "/chronograf/v1/users"
+					  },
+					  "users": [
+					    {
+					      "links": {
+					        "self": "/chronograf/v1/users/1"
+					      },
+					      "id": "1",
+					      "name": "billibob",
+					      "provider": "github",
+					      "scheme": "oauth2",
+					      "superAdmin": false,
+					      "roles": [
+					        {
+					          "name": "admin",
+					          "organization": "0"
+					        }
+					      ]
+					    }
+					  ]
+					}`,
+			},
+		},
+		{
+			name:    "GET /users",
+			subName: "Single User in the Default Organization as Editor",
```

Comment:
same as above -- is it possible to have only one user in Default who is an Editor?